docs/AboutUs.md

```diff
@@ -4,49 +4,36 @@ We are a team based in the [School of Computing, National University of Singapor
 
 ## Project Team
 
-#### [Damith C. Rajapakse](http://www.comp.nus.edu.sg/~damithch) <br>
-<img src="images/DamithRajapakse.jpg" width="150"><br>
+#### [Chuan Wei Candiie](https://github.com/Candiie) <br>
+<img src="images/Candiie.png", width="150"><br>
 **Role**: Project Advisor
 
------
+----- 
 
-#### [Joshua Lee](http://github.com/lejolly)
-<img src="images/JoshuaLee.jpg" width="150"><br>
-Role: Developer <br>  
-Responsibilities: UI
+#### [Lee Wenwei Johnervan](http://github.com/johnervan) <br>
+<img src="images/JohnervanLee.png", width="150"><br>
```

Comment:
Super awesome photos! Very Professional looking! (Do show it during presentation if time permits XD) Wait! @weikangchia @jaeoheeail @johnervan @origiri  did you guys take them together?! the background are too similar to have been taken separately.
👍